plugins/languages/nvim-jdtls.nix

```diff
@@ -105,7 +105,7 @@ in {
           else cfg.data;
       in
         [
-          "${pkgs.jdt-language-server}/bin/jdt-language-server"
+          "${lib.getExe pkgs.jdt-language-server}"
```

Comment:
Why use string templating here instead of an expression?

```suggestion
          (lib.getExe pkgs.jdt-language-server)
```

Reply:
I agree, this is what I had suggested at first.
But I'm trying to stop nitpicking !

Reply:
Sorry, I missed that in your original review. I just glanced at the suggested change from my phone and thought it was just the lib reference.

Reply:
No worry, I fixed it :)